HEX Here 2 GPS: usage instructions (#2475)
PX4/PX4-user_guide: en/gps_compass/gps_hex_here2.md

> If the GPS is *not detected* then [update the Here2 firmware](https://docs.cubepilot.org/user-guides/here-2/updating-here-2-firmware).
> :::
> 
> :::note
> If the GPS is detected but not works,  then try to [allocating node uavcan ID method](https://docs.cubepilot.org/user-guides/here-2/here-2-can-mode-instruction).

Reviewer: ```suggestion
- If the GPS is *not detected* then [update the Here2 firmware](https://docs.cubepilot.org/user-guides/here-2/updating-here-2-firmware).
- If the GPS is detected but does not work, attempt the process outlined in [allocating node uavcan ID](https://docs.cubepilot.org/user-guides/here-2/here-2-can-mode-instruction).
```
